projects/caddy/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/caddyserver/caddy"
+main_repo: "https://github.com/caddyserver/caddy"
+primary_contact: ""
+auto_ccs :
+  - "adam@adalogics.com"
```

Comment:
Can you please add my email address too?

Reply:
Added!